examples/cms/src/data.ts

```diff
@@ -0,0 +1,150 @@
+import { CollectionItemData, ManagedCollection, ManagedCollectionField, framer } from "framer-plugin"
+import { PLUGIN_KEYS } from "./constants"
+
+export type ManagedCollectionFieldType = ManagedCollectionField["type"]
+
+type DataSourceField =
+    | {
+          type: ManagedCollectionFieldType
+      }
+    | {
+          type: "html"
+      }
```

Comment:
With Vite, would it be possible to get these types from the JSON directly without having to manually specifying these?

Reply:
I don't think so as we will need to import (instead of fetching) the JSON here